ctms/schemas/contact.py

```diff
@@ -192,15 +198,40 @@ class ContactPatchSchema(ComparableBase):
     "UNSUBSCRIBE" instead of lists or objects.
     """
 
-    amo: Optional[Union[Literal["DELETE"], AddOnsInSchema]]
-    email: Optional[EmailPatchSchema]
-    fxa: Optional[Union[Literal["DELETE"], FirefoxAccountsInSchema]]
-    mofo: Optional[Union[Literal["DELETE"], MozillaFoundationInSchema]]
-    newsletters: Optional[Union[List[NewsletterSchema], Literal["UNSUBSCRIBE"]]]
-    waitlists: Optional[Union[List[WaitlistInSchema], Literal["UNSUBSCRIBE"]]]
-    # TODO waitlist: remove once Basket leverages the `waitlists` field.
-    vpn_waitlist: Optional[Union[Literal["DELETE"], VpnWaitlistInSchema]]
-    relay_waitlist: Optional[Union[Literal["DELETE"], RelayWaitlistInSchema]]
+    amo: Optional[Union[Literal["DELETE"], AddOnsInSchema]] = Field(
```

Comment:
I'm new-ish to typing, but I'm curious why not the newer syntax of:
```python
    amo: Literal["DELETE"] | AddOnsInSchema | None = Field(
```

Reply:
Yeah I think that'd work too! For this PR, I was focused on doing as little as possible (which was still _a lot_ to make things v2 compatible and not touch much else).

I think doing things like refining these types would be good for a future PR.